src/components/AttachmentPicker/index.native.js

```diff
@@ -111,13 +121,15 @@ class AttachmentPicker extends Component {
 
         this.close = this.close.bind(this);
         this.pickAttachment = this.pickAttachment.bind(this);
+        this.completeAttachmentSelection = () => {};
```

Comment:
I'd say we should not add this for now, but make a separate issue to clean up `completeAttachmentSelection` - as you said, it looks pretty horrible in this file, but that's not the point of this PR

Reply:
@akshayasalvi Could you please remove this line based on the request above?